src/sentry/options/defaults.py

```diff
@@ -128,6 +128,9 @@
 
 
 register("api.rate-limit.org-create", default=5, flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK)
+register(
```

Comment:
Are you trying to provide a way for someone to change the limit for this endpoint without a deploy? This wouldn't do that, AFAIK.
I don't think we have a tested way today to modify rate limits on-the-fly.

Reply:
I don't think we need to be able to change the rate limits, based on https://sentry.slack.com/archives/CG297KGHM/p1657296789127049